doc/getting_started/installation_linux.rst

```diff
@@ -52,23 +52,32 @@ Install the required packages in a Ubuntu host system with:
 
 .. code-block:: console
 
-   $ sudo apt-get install git make gcc g++ ncurses-dev gperf ccache\
-	 doxygen dfu-util device-tree-compiler python3-ply python3-pip
+   $ sudo apt-get install --no-install-recommends git ninja-build gperf \
```

Comment:
Not all these changes for installed files are cmake related?

Reply:
Right.

The Ubuntu apt install command has changed slightly.

We use --no-install-recommends to have more control over exactly which packages we depend on.

This was tested using Docker, which uses a minimal Ubuntu install. This new apt-get install 
command is safer than the old one because it is explicitly listing some dependencies that
were assumed before.